src/telliot_feeds/feeds/twap_manual_feed.py

```diff
@@ -0,0 +1,14 @@
+from telliot_feeds.datafeed import DataFeed
+from telliot_feeds.queries.price.twap import TWAP
+from telliot_feeds.sources.manual_sources.twap_manual_input_source import TWAPManualSource
+
+
+# Using defaults to bypass an error handle for auto feeds and unsupported currencies
+asset = "eth"
+currency = "usd"
```

Comment:
Idk if I've asked about this before on other PRs, but is there a test that ensures these get updated when this feed is used with the `--build-feed` flag or in other contexts?

Reply:
there is a cli test for build a feed: https://github.com/tellor-io/telliot-feeds/blob/e8cad3427a99b69cc05be3d7dc7b1653dd0502b6/tests/cli/test_cli.py#L24